pages/interface/components/ReadTime/index.js

```diff
@@ -0,0 +1,14 @@
+import { Text } from '@/TabNewsUI';
+
+function getReadTime(text, wpm = 260) {
+  const wordsQuantity = text.split(' ').length;
```

Comment:
Provavelmente performance não seria um problema aqui mas se tu quiser deixar mais rápido:

- Usa a lib [alfaaz](https://github.com/thecodrr/alfaaz), 2x mais rápido que split
- Ou troca o `.split` por um simples for, isso consumiria menos memória já que não precisa alocar para dividir o texto inteiro.